### ISSUE TITLE
Static Calendar Color Contrast

### ISSUE THREAD

**Issue by pancakereport** [berkeley-cdss/berkeley-class-site#83]
static calendar doesn't meet color contrast standards by default (newly named event). see [here](https://github.com/data-8/fa25/commit/ad7406c945af177201d9c57d6851cd21cd95597b) for an example
